Aptoide/aptoide-client-v8
# [MOB-692] Mitigate pop up experience packageinstaller

**What does this PR do?**

This PR aims at using package installer on the updates. We also took the chance to start using this, not only for the updates, but for all apps and all installs (appc or non appc). The only exception are xiaomi devices that dont have the aab fix. The devices lower than 21 were also changed to use this, since package installer also supports their <21 installation.

Also pulled the PackageInstallerManager to main folder, since the requirements were equal for all flavours.
Also removed monetisation flavour as it is not needed anymore.

**PS: DO NOT ACCEPT THIS UNTIL RELEASE V9.16.0.1 IS ACCEPTED.** ( the app was crashing on devices < 21 and this was fixed for that release, so I had to merge the code from that release.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

Open the app and try several installations, both above 21 and below. Also try updates and update all to make sure everything is ok. 

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-692](https://aptoide.atlassian.net/browse/MOB-692)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**
